Gems/EMotionFX/Code/EMotionFX/Tools/EMotionStudio/EMStudioSDK/Source/MainWindow.cpp

```diff
@@ -577,6 +580,8 @@ namespace EMStudio
         AZ_Assert(!m_nativeEventFilter, "Double initialization?");
         m_nativeEventFilter = new NativeEventFilter(this);
         QAbstractEventDispatcher::instance()->installNativeEventFilter(m_nativeEventFilter);
+
+        EnableUpdatingPlugins();
```

Comment:
Hmm.. this doesn't make much sense. Why connect back to the TickBus at the end of destruction?

Reply:
Expand the view, this is just GitHub squashing the code I think. This is at the end of the constructor.